src/Sarif/NotYetAutogenerated/SuppressionStates.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft.  All Rights Reserved.
+// Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.CodeDom.Compiler;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    /// <summary>
+    /// The state of a result relative to a baseline of a previous run.
+    /// </summary>
+    [GeneratedCode("Microsoft.Json.Schema.ToDotNet", "0.16.0.0")]
+    [Flags]
+    public enum SuppressionStates
+    {
+        None,
```

Comment:
None means we have no SuppressionState information of any kind. None is a typical name for an enum zero value